float64 waypointAcceptanceRadii
float64 nfzBufferSize